src/components/Spinner/Spinner.tsx

```diff
@@ -35,33 +23,14 @@ export interface SpinnerProps {
 
 export function Spinner({
   size = 'large',
-  color = 'teal',
```

Comment:
Over in https://github.com/Shopify/polaris-react/issues/3487 there was talk of needing colors on a Spinner. Thoughts on keeping this hanging around for the moment?